### PR TITLE
SSR guide (archived) - Small updates

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Update notes about availability of feature on Classic. Update details about password policy.
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-650540](https://oktainc.atlassian.net/browse/OKTA-650540)
* [OKTA-721801](https://oktainc.atlassian.net/browse/OKTA-721801)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md](https://github.com/okta/okta-developer-docs/blob/4abd1fe210cab8a84e93c3172735c4b9b3f398d8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Farchive-set-up-self-service-registration%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/be770bce-02da-43cb-9fae-c0dd66fe2c38) | 56  | 79  | 86  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4ef14350-37f0-49f5-9cd8-50ddf92358fe?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md [hunk 1]
```diff
@@ -2,43 +2,46 @@
...
+   > **Note:** The **Activation emails are valid for** element sets the lifetime of the emailed link. Find it on the **Security** > **General** page of the Admin Console.
 
 * **Make email verification optional.** After registration, users are immediately redirected to their custom app/portal or to their org's Okta homepage. Okta then sends the user an email to request verification of their email address.
 
 ## Enable and configure a self-service registration policy
 
-Use a self-service registration policy to let users self-register for application access. When you create the policy, you can choose the fields that you want to include on the Create Account registration form. You can also specify how those fields are ordered, and mark which of the fields are required.
+Use a self-service registration policy to let users self-register for application access. When you create the policy, you can choose the fields that you want to include on the Create Account registration form. You can also specify the order of the fields, and the fields that are required.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md:44]:
```suggestion
Use a self-service registration policy to let users self-register for app access. When you create the policy, you can choose the fields that you want to include on the Create Account registration form. You can also specify the order of the fields, and the fields that are required.
```

File: packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md [hunk 2]
```diff
@@ -48,8 +51,10 @@ When you enable a self-service registration policy, Okta enforces uniqueness for
...
 
-   * **Add to Sign-In Widget:** Optional. Select this option if you want to add a sign-up link to your Okta-hosted sign-in page. By adding a sign-up link to Okta Sign-In Widget, you eliminate the need to configure a link that uses JavaScript in the Customize Sign-In Page editor.
-   * **Assign to group:** Optional. Enter the existing group name that users are automatically added to when they self-register. Group membership determines which password policy is applied during registration. If no group is specified, Okta applies the Default password policy.
+   * **Add to Sign-In Widget:** Optional. Select this option if you want to add a sign-up link to your Okta-hosted sign-in page. By adding a sign-up link to the Okta Sign-In Widget, you don't need to configure a link that uses JavaScript in the editor.
+   * **Assign to group:** Optional. Enter the group name that users are automatically added to when they self-register. Group membership determines which password policy is applied during registration. If no group is specified, Okta applies the default password policy even if that policy is applied for non-Okta sourced users.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md:55]:
```suggestion
   * **Assign to group:** Optional. Enter the group name that users are automatically added to when they self-register. Group membership determines which password policy is applied during registration. If no group is specified, Okta applies the default password policy even if that policy is applied for non-Okta-sourced users.
```

Comment (barbaravo-okta) [reply packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md:55]: I mean, the hyphen is correct, but now it looks like a word centipede. I'm good with whatever you think is best.

File: packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md [hunk 3]
```diff
@@ -48,8 +51,10 @@ When you enable a self-service registration policy, Okta enforces uniqueness for
...
+
+   > **Note:** If you're using Identity Engine, Okta doesn't apply the default password policy to non-Okta sourced users. See [Self-service registration](/docs/guides/oie-embedded-sdk-use-case-self-reg/android/main/).
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md:57]: Same deal here

File: packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md [hunk 4]
```diff
@@ -76,7 +81,9 @@ When you enable a self-service registration policy, Okta enforces uniqueness for
 
 1. In the **POST REGISTRATION** section, complete these fields:
 
-    * **Activation requirements:** Optional. Select the checkbox to automatically send users a register activation email. If you don't select the checkbox, and users aren't required to verify their email address for activation, a Registration Verification email that uses the Okta email template is sent.
+    * **Activation requirements:** Optional. Select the checkbox to automatically send users a register activation email. A Registration Verification email is sent, using the Okta email template, given the following:
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md:84]:
```suggestion
    * **Activation requirements:** Optional. Select the checkbox to automatically send users a register activation email. A registration verification email is sent, using the Okta email template, given the following:
```

File: packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md [hunk 5]
```diff
@@ -86,35 +93,49 @@ When you enable a self-service registration policy, Okta enforces uniqueness for
    > **Note:** Ensure that the domain for the custom URL that you've configured for the **Default redirect** is added as a [Trusted Origin](/docs/guides/enable-cors/) for redirects.
 
 1. Click **Save**.
-1. Optional. If you added more fields (existing Okta user profile attributes) from the attribute list to the registration form during step 4, and you want to allow your users to modify those fields after registration on their **Settings** page:
+1. Optional. Follow these steps given the following:
+
+  * You can add more fields (existing Okta user profile attributes) from the attribute list to the registration form
+  * You want to allow users to modify those fields after registration on their **Settings** page
+
+    1. Go to **Directory** and then **Profile Editor**.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md:101]:
```suggestion
    1. Go to **Directory** > **Profile Editor**.
```

**Review by barbaravo-okta — CHANGES_REQUESTED** [review]
Just a few tiny nits

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md](https://github.com/okta/okta-developer-docs/blob/18f36f963e6046c0a0ba3fdbc30cf8faecb083c3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Farchive-set-up-self-service-registration%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/030b545d-5e50-4a70-8147-7b43c63a37cd) | 56  | 79  | 86  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e9066480-7632-45d2-b629-85bc511801f2?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md](https://github.com/okta/okta-developer-docs/blob/c3ac4b5745874424109c2893f2fb37a6d183dce8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Farchive-set-up-self-service-registration%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1475a646-00b5-44d0-aa51-5b55cd133202) | 56  | 79  | 86  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.ab976f3b-ca88-42a7-b11d-608c42009dff?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md](https://github.com/okta/okta-developer-docs/blob/e8659796ab7f0b1b52074f156d7de9cf812037f2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Farchive-set-up-self-service-registration%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b9b8630a-1a86-4f8d-b1d7-c90afb0d1c63) | 56  | 79  | 86  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.71e07462-7439-4d36-b255-a48f4e2bb5f4?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md](https://github.com/okta/okta-developer-docs/blob/24131f21fe103bb4a7e8c5b80a98cbc4802d811c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Farchive-set-up-self-service-registration%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8a215460-0878-43a0-9b16-ad52a3bb8d61) | 56  | 79  | 86  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e181a5af-5657-4db8-aea4-17cc31244361?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/archive-set-up-self-service-registration/main/index.md](https://github.com/okta/okta-developer-docs/blob/39e934d0b9bed5376d164afd70373459652d6ee6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Farchive-set-up-self-service-registration%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/67a7e217-dac6-4aa4-ae62-0d25c8763f27) | 56  | 79  | 86  | 99  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.52aa65b8-a633-45f5-888a-5a7523e446de?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**